<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashBoard</class>
 <widget class="QMainWindow" name="DashBoard">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Independent National Electoral Commission - OpenVR</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="devPrinter">
    <property name="geometry">
     <rect>
      <x>647</x>
      <y>380</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devFP">
    <property name="geometry">
     <rect>
      <x>647</x>
      <y>136</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devWebcam">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>256</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>406</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Printer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>162</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fingerprint Scanner</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>784</x>
      <y>282</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Webcam</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="devRemovable">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>13</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid grey; background-color:rgb(255, 196, 196);padding:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>784</x>
      <y>44</y>
      <width>210</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Backup Storage</string>
    </property>
   </widget>
   <widget class="QLabel" name="leFP">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>192</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect FingerPrint</string>
    </property>
   </widget>
   <widget class="QLabel" name="lePrinter">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>430</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Printer</string>
    </property>
   </widget>
   <widget class="QLabel" name="leRemovable">
    <property name="geometry">
     <rect>
      <x>784</x>
      <y>70</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Backup</string>
    </property>
   </widget>
   <widget class="QLabel" name="leWebcam">
    <property name="geometry">
     <rect>
      <x>786</x>
      <y>310</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect Webcam</string>
    </property>
   </widget>
   <widget class="QLabel" name="devPrinter_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/hp-printer.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devFP_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>132</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/fp-scanner.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devRemovable_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/usb-flash-drive.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="devWebcam_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/webcam.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OpenVR</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>431</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main Menu</string>
     </attribute>
     <widget class="QCommandLinkButton" name="btnShutdown">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>320</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Shutdown</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/lock.png</normaloff>resources/icons/lock.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnReports">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reports</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/chart.png</normaloff>resources/icons/chart.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>81</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F3]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnVotersCard">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>73</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Print Voter's Card</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/voter-card.png</normaloff>resources/icons/voter-card.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>186</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F5]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnQueueTickets">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>124</y>
        <width>231</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Queue Tickets </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/ticket-icon.png</normaloff>resources/icons/ticket-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnRegistration">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active/>
        <inactive/>
        <disabled/>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Registration</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/camera.png</normaloff>resources/icons/camera.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>36</width>
        <height>48</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>320</y>
        <width>51</width>
        <height>38</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F12]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>30</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F2]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>136</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F4]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnBackup">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DDC Backup</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/Arrow Down.png</normaloff>resources/icons/Arrow Down.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>231</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F6]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>279</y>
        <width>36</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>[F7]</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="btnControlPanel">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>268</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Control Panel</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/icons/control-panel.png</normaloff>resources/icons/control-panel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Technical Support</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>About</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
